docs/source/reference-overview/security/security-overview.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Security Reference
+navigation_title: "Security"
+---
+
+% TO-DO: Add links to "What is Elastic Security?"%
+
+This section of the documentation contains reference information for Elastic Security features, including:
+
+* Prebuilt rules
+* Downloadable rule updates
+* Prebuilt jobs
+* Fields and object schemas
+
+You can use these APIs to interface with Elastic Security features:
+
+* Detections API: Manage detection rules and alerts
+* Exceptions API: Create and manage rule exceptions
+* Lists API: Create source event value lists for use with rule exceptions
+* Timeline API: Import and export timelines
+* Cases API: Open and manage cases
+* Elastic AI Assistant API: Interact with and manage Elastic AI Assistant
+* Asset criticality API: Create and manage asset criticality records
+
+Additionally, the Kibana Actions API is partially documented to enable opening and updating cases in external ticketing systems. For more information on Kibana Actions, see Alerting and Actions and action plugins.
```

Comment:
> Additionally, the Kibana Actions API is partially documented to enable opening and updating cases in external ticketing systems. For more information on Kibana Actions, see Alerting and Actions and action plugins.

IMO This blurb isn't worth keeping, since there's no security-specific action APIs (it's just the shared Kibana connector APIs https://www.elastic.co/docs/api/doc/kibana/group/endpoint-connectors). The overall concept of alerting will be covered in another setion of the nav (URL TBD) so that link can be reconsidered later.

Reply:
makes sense, I'll remove it 👍